content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -552,6 +551,11 @@ If you want to reset the IPVS tables, you must run the following command:
 ipvsadm -C
 ```
 
+Now remove the node:
+```
```

Comment:
@vinu2003 can you mark this as bash code, to match the other blocks?

Ideally, amend the existing commit and then force-push it.

Reply:
Thanks for pointing this out @sftim